pages/phases/assessments/notifications.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: article
-permalink: /hiring-phases/phone-assessment-interviews/notifications/
+permalink: /hiring-phases/assessments/notifications/
 section: hiring-phases
 title: Notifying Applicants After Interviews
 sidenav: hiring-phases
-phase: phone-assessment-interviews
+phase: assessments
 sub-phase-order: 040
 intro: After SMEs have completed interviews, you should send notifications to all applicants indicating their status. Use these NOR codes and notification messages to communicate with applicants.
```

Comment:
it seems like we're still implying there must be an interview after the demonstration project/written assessment. is that right?

Reply:
Good catch. I didn't update this and have here https://github.com/usds/SME-QA-Website/pull/243/commits/497f103a4cf0976fd94c66bf4076279a1a78d9a0